tests/foreman/ui/test_settings.py

```diff
@@ -394,7 +394,9 @@ def test_positive_update_email_delivery_method_sendmail(session, target_sat):
         "send_welcome_email": "Yes",
     }
     command = "grep " + f'{mail_config_new_params["email_subject_prefix"]}' + " /var/mail/root"
-
+    result = target_sat.execute('systemctl status postfix')
+    if "Active: inactive (dead)" in result.stdout or result.status != 0:
+        target_sat.execute('systemctl restart postfix')
```

Comment:
Instead of matching the exact string you can also try this.
```suggestion
    if target_sat.execute('systemctl status postfix').status!=0:
          target_sat.execute('systemctl restart postfix')
```

Reply:
> This change looks good to me. I tested it locally and it is passing there, so I'm not sure why it's failing in CI. Looking at robottelo.log from the PRT run, stdout and stderr are both empty when postfix.service is restarted, which suggests that the restart was successful. I'm going to kick off another PRT run; if the error persists, it may be helpful to debug this on a running robottelo container in CI to determine the cause.

Hey @synkd I see the very first PRT run passed without any issues not sure what went wrong for the last two runs

Reply:
Ack @adarshdubey-star, good catch. I'm not sure what went wrong either. I do see that the failed runs also failed dynaconf validation, but I don't think that would cause this particular issue. It seems to be failing consistently on line 406, though. I see you just kicked off another PRT run, so let's see if it remains consistent.